README.md

```diff
@@ -1,3 +1,7 @@
+# Disclaimer
+
+**This repository is community supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including Third Party Integrations and Mattermost Integrations. Integrations may be modified or discontinued at any time.**
```

Comment:
0/5 on if this is correct. Maybe not worth changing if legal needs to revisit the wording

```suggestion
**This repository is community-supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including third-party integrations and Mattermost integrations. Integrations may be modified or discontinued at any time.**
```

> Mattermost disclaims liability for ... Mattermost integrations

Is this true?

Reply:
This wording has been provided from Legal and should not be modified.